src/Sentry/Laravel/Integration.php

```diff
@@ -122,27 +123,45 @@ public static function flushEvents(): void
      * @param \Illuminate\Routing\Route $route
      *
      * @return string
+     *
+     * @deprecated This will be removed in version 3.0, use `extractNameAndSourceForRoute` instead.
```

Comment:
Yes, since it's public it't technically a breaking change if we just remove it or change the return type. I know of at least 1 person using it in their apps too 👀

Reply:
Maybe we should slap an `@internal` on to this at one point.

Reply:
That's actually a good idea yeah! Lemme do that real quick!